applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -0,0 +1,1552 @@
+
+   #. Start receiving and acknowledging the data.
+      Replace *example.com* with the host name or IPv4 address of the UDP client and *1234* with the corresponding port.
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT#XRECVFROM**
+         Hello, UDP#1!
+         #XRECVFROM: 1, 13
+         OK
+
+         **AT#XRECVFROM**
+         Hello, UDP#2!
+         #XRECVFROM: 1, 13
+         OK
+
+         **AT#XSENDTO="**\ *example.com*\ **",**\ *1234*\ **,1,"UDP1/2 received"**
+         #XSENDTO: 15
+         OK
+
+         **AT#XRECVFROM**
+         Hello, UDP#3!
+         #XRECVFROM: 1, 13
+         OK
+
+         **AT#XRECVFROM**
+         Hello, UDP#4!
+         #XRECVFROM: 1, 13
+         OK
+
+         **AT#XRECVFROM**
+         Hello, UDP#5!
+         #XRECVFROM: 1, 13
+         OK
+
+         **AT#XSENDTO="**\ *example.com*\ **",**\ *1234*\ **,1,"UDP3/4/5 received"**
+         #XSENDTO: 17
+         OK
+
+      Note that you will get an error message if a UDP packet is lost.
+      For example, this error indicates that a packet is lost in the downlink to the nRF9160 DK:
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT#XRECVFROM**
+         #XSOCKET: -60
+         ERROR
+
+   #. Observe the output of the Python script::
+
+         $ python client_udp.py
+
+         Sending: 'Hello, UDP#1!
+         Sending: 'Hello, UDP#2!
+         UDP1/2 received
+         ('000.000.000.00', 1234)
+         Sending: 'Hello, UDP#3!
+         Sending: 'Hello, UDP#4!
+         Sending: 'Hello, UDP#5!
+         UDP3/4/5 received
+         ('000.000.000.00', 1234)
+         Closing connection
+
+   #. Close the socket.
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT#XSOCKET=0**
+         #XSOCKET: 0, closed
+         OK
+
+#. Test the UDP server with UDP proxy service (note that these commands are available only if :option:`CONFIG_SLM_UDP_PROXY` is defined):
+
+   a. Check the available values for the XUDPSVR command and create a UDP server.
+      Replace *1234* with the correct port number.
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT#XUDPSVR=?**
+         #XUDPSVR: (0, 1, 2),<port>,<sec_tag>
+         OK
+
+         **AT#XUDPSVR=1,**\ *1234*
+         #XUDPSVR: 2 started
+         OK
+
+   #. Run the :file:`client_udp.py` script to start sending data to the server.
+
+   #. Observe that the server starts receiving data and acknowledge the data.
+
+      .. parsed-literal::
+         :class: highlight
+
+         #XUDPRECV: 1, 13
+         Hello, UDP#1!
+         #XUDPRECV: 1, 13
+         Hello, UDP#2!
+
+         **AT#XUDPSEND=1,"UDP1/2 received"**
+         #XUDPSEND: 15
+         OK
+
+         #XUDPRECV: 1, 13
+         Hello, UDP#3!
+         #XUDPRECV: 1, 13
+         Hello, UDP#4!
+         #XUDPRECV: 1, 13
+         Hello, UDP#5!
+
+         **AT#XUDPSEND=1,"UDP3/4/5 received"**
+         #XUDPSEND: 17
+         OK
```

Comment:
The word document has a second series of 
#XUDPRECV: 1, 13
Hello, UDP#1!
etc.

Should that be here, or is it just a mistake?

Reply:
Not a mistake, UDP client ran twice.
But you can remove the second series